rest/model/user_test.go

```diff
@@ -0,0 +1,32 @@
+package model
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/model/user"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserSettings(t *testing.T) {
```

Comment:
Add a test that validates the empty settings case/partial settings case.

Reply:
added